src/Components/AboutPage.js

```diff
@@ -123,25 +130,32 @@ const CtaP = styled.p`
   margin-bottom: 30px;
 `;
 
-const BackgroundImgContainer = styled.div`
-  width: 100vw;
-  height: 200vh;
-  background-image: url("../assets/bannerImages/backgroundImage1.png");
-  background-position: center;
-  background-repeat: no-repeat;
-  background-size: cover;
-  background-attachment: fixed;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
+const AboutContainer = styled.div`
+  height: 100vh;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  perspective: 10px;
+  margin-bottom: 20px;
 `;
+
+const ParallaxImage = styled.img`
```

Comment:
Super cool! Great work getting that figured out. I'm going to have to look into this more.